src/pages/Faucet/index.tsx

```diff
@@ -38,23 +38,35 @@ export const FaucetPage = () => {
     },
     [address]
   );
+
   const onModalClose = () => {
     setIsModalOpen(false);
     setTxHash("");
   };
+
+  const ReadEthAmount = () => {
```

Comment:
I don't think we should be breaking our pattern for hook naming convention - please extract this to its own hook called `useFaucetBalance` - or, just call `useBalance` and reference the `data` constant in the jsx